apps/src/templates/sectionProgress/SectionProgress.jsx

```diff
@@ -131,12 +136,16 @@ class SectionProgress extends Component {
       currentView,
       scriptId,
       scriptData,
+      sectionId,
       showStandardsIntroDialog,
     } = this.props;
     const levelDataInitialized = this.levelDataInitialized();
     const lessons = scriptData ? scriptData.lessons : [];
     const scriptWithStandardsSelected =
       levelDataInitialized && scriptData.hasStandards;
+    const progressTableViewEnabled =
```

Comment:
After reading the rest of the file I understand what this means (updating the variable name to 'enabled' is more clear than before) but it still feels a little nonintuitive. Would something more ui-based like 'showProgressTable' make sense?

Reply:
Sure that's a cleaner name. I'll make that change.